Makefile

```diff
@@ -2,7 +2,7 @@
 
 # nice way to have our .env in environment for use in makefile
 # see https://lithic.tech/blog/2020-05/makefile-dot-env
-# Note: this will mask environment variable as opposed to docker-compose priority
+# Note: this will mask environment variable as opposed to docker compose priority
```

Comment:
why have you introducted these changes?

Reply:
Same thing, I made that because on my computer for unknown reason the "docker compose" don't exists, I have to use "docker-compose". This should be removed.